transcom/mymove
# B 19843 adjust ssw text to say 'claimed expensed'

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19843)

### How to test

1. run the `db_dev_e2e_populate` or build a ppm with closeout and an advance.
2. download the aoa packet from the move page, or move entry on the main service member dashboard.
3. You should see 'Claimed' instead of 'Claimable'  Expenses.

## Screenshots
